Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/random_perturbation.h

```diff
@@ -112,6 +110,8 @@ namespace internal {
 *    constrained-or-not status of each vertex of `tmesh`. A constrained vertex
 *    cannot be modified at all during remeshing
 *  \cgalParamEnd
+*  \cgalParambegin{random_seed} an non-negative integer value to seed the random
```

Comment:
"non-negative" means `>=0`. What would you say othewise? "Positive" means `>0`...